pennylane/ops/channel.py

```diff
@@ -644,13 +686,15 @@ class QubitChannel(Channel):
     Args:
         K_list (list[array[complex]]): list of Kraus matrices
         wires (Union[Wires, Sequence[int], or int]): the wire(s) the operation acts on
+        do_queue (bool): Indicates whether the operator should be
+            immediately pushed into the Operator queue (optional)
+        id (str or None): String representing the operation (optional)
     """
     num_wires = AnyWires
     grad_method = None
 
-    def __init__(self, *params, wires=None, do_queue=True):
-        super().__init__(*params, wires=wires, do_queue=do_queue)
-        K_list = params[0]
+    def __init__(self, K_list, wires=None, do_queue=True, id=None):
```

Comment:
Is `wires` being a keyword argument here intentional? Mostly you seemed to have added wires as a positional argument, and the current documentation has it positional as well.

(there are a few other instances like this throughout)

Reply:
Good question, I think for pretty much every case we would require a specific value for the wires, so it should really be a keyword argument everywhere. But most of the operations have it as a positional argument, so I tried to stay consistent with what ever convention the old ``__init__`` method used.

Reply:
I would seize this opportunity to ensure it is consistent across all operations.
Given that all docstrings have reported it as positional so far, I would recommend going with that.

Reply:
I would check in with @mariaschuld here: I think there is a long-term aim to allow for `wires=None`, so that non-circuit operator arithmetic can be performed easily. I'm not sure though if this example is an historical accident, or a recent change.